src/components/Button/Button.spec.tsx

```diff
@@ -1,14 +1,79 @@
 import * as React from 'react';
-import {Button} from './';
+import defaults = require('lodash/defaults');
+import {createDriverFactory} from 'wix-ui-test-utils/driver-factory';
+import { buttonDriverFactory } from "./Button.driver";
 import {buttonTestkitFactory} from '../../testkit';
 import {buttonTestkitFactory as enzymeButtonTestkitFactory} from '../../testkit/enzyme';
 import {runTestkitExistsSuite} from '../../common/testkitTests';
+import {Button,ButtonProps} from './';
+import {Skin, Priority, Size} from './constants';
+import {enumValues} from '../../utils';
+
 
 describe('Button', () => {
+  const createDriver = createDriverFactory(buttonDriverFactory);
+  const PButton = partialHOC(Button, 
+    {
+      children: 'Click me!'
+    });
+
+  describe('skin prop', () => {
+    it(`should be '${Skin.standard}' by default`, () => {
```

Comment:
I know it is better to have one source of truth that even the test description will have, but actually in practice I found out that it is annoying that once you search for the failing test you won't find it with cmd + f

Reply:
changed